jest.config.js

```diff
@@ -112,7 +112,7 @@ module.exports = {
   // setupFiles: [],
 
   // A list of paths to modules that run some code to configure or set up the testing framework before each test
-  // setupFilesAfterEnv: [],
+  setupFilesAfterEnv: ['./test/setup.js'],
```

Comment:
@jgerigmeyer Turns out this wasn't a good use case for `globals` setting; I was getting weird behavior where `BluebirdPromise` ended up being `undefined` if I tried to set it in `globals`. I'm not sure why but it says in the docs that `globals` has to be JSON-serializable, which I suspect might be the problem. So I just switched to a setup file.